src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -141,6 +142,11 @@ public sealed partial class QuicStream : System.IO.Stream
         public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override void WriteByte(byte value) { }
     }
+    public readonly partial struct QuicStreamCapacityChangedArgs
+    {
```

Comment:
```suggestion
    {
        private readonly object _dummy;
        private readonly int _dummyPrimitive;
```
(this is needed for structs in `ref` assemblies to prevent the caller from doing weird stuff with structs that appear to have no fields)

Reply:
Both of them? Asking because ref source generator generates only the `int` one.

Reply:
Yes, including the object gives us the option to safely add reference fields in the future.

As an example, without the dummy object field, someone could theoretically write this without having to suppress warnings about creating a pointer to a managed type:
```c#
QuicStreamCapacityChangedArgs* ptr = &val;
```

Reply:
Done.